include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -4059,8 +4159,9 @@ MGTwoLevelTransferNonNested<dim, LinearAlgebra::distributed::Vector<Number>>::
 
   const auto evaluation_function = [&](const auto &values,
                                        const auto &cell_data) {
-    std::vector<Number>                    solution_values;
-    FEPointEvaluation<1, dim, dim, Number> evaluator(*mapping_info, *fe_coarse);
+    std::vector<Number>                               solution_values;
+    FEPointEvaluation<n_components, dim, dim, Number> evaluator(*mapping_info,
+                                                                *fe_coarse);
```

Comment:
We might also need a base class of `FEPointEvaluation` that is not templated with `n_components`. Alternatively, what the nested implementation does is not to use `FEEvaluation` but directly the tensor product kernels. But I am not sure if this is the right approach here.